projects/demo/src/modules/components/combo-box/examples/2/index.html

```diff
@@ -1,46 +1,50 @@
-<tui-combo-box
-    *tuiLet="service.request(search) | async as filtered"
-    class="b-form"
-    [formControl]="control"
-    [valueContent]="value"
-    [(search)]="search"
->
+<label>
     Find the best employees
-    <input
-        placeholder="Type a name"
-        tuiTextfield
-    />
-    <ng-template
-        #value
-        let-item
+
+    <tui-combo-box
+        *tuiLet="service.request(search) | async as filtered"
+        class="b-form"
+        [formControl]="control"
```

Comment:
Should have labelOutside enabled and internal label removed. Inspect other examples too please so they look natural.

Reply:
fixed